ee/vellum_ee/workflows/tests/test_serialize_module.py

```diff
@@ -1,6 +1,17 @@
 from vellum_ee.workflows.display.workflows.base_workflow_display import BaseWorkflowDisplay
 
 
+def test_serialize_module_with_dataset():
```

Comment:
add another test where we actually had a `sandbox.py` defined so that we see the content in `dataset`